pkgs/tools/system/gdu/default.nix

```
@@ -6,21 +6,21 @@
 
 buildGoModule rec {
   pname = "gdu";
-  version = "2.3.0";
+  version = "3.0.0";
 
   src = fetchFromGitHub {
     owner = "dundee";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1dc0z6daxpbid2ilpms0dw76qyyc84gx0bcqsx0b3s5p5p154xzq";
+    sha256 = "0sfb8bxvdd8r05d0bgfcaw6dpbky7f4fgf0dbly7k7sgl29hkafy";
   };
 
-  vendorSha256 = "18a3qwshz8jmw0j29qvmzarxig0kj1n0fnmlx81qzswsyl85kncv";
+  vendorSha256 = "0w3k23kly8g9mf8a300xz6bv7g1m2nlp5f112k4viyi9zy6vqbv0";
 
-  buildFlagsArray = [ "-ldflags=-s -w -X main.AppVersion=${version}" ];
+  buildFlagsArray = [ "-ldflags=-s -w -X github.com/dundee/gdu/build.Version=${version}" ];
 
-  # analyze/dev_test.go: undefined: processMounts
-  doCheck = !stdenv.isDarwin;
+  # tests fail is the version is set
+  doCheck = false;
```

`TestVersion` is using `development`. Looks like that the tests could pass by replacing the `development` with the current version.

```suggestion
  postPatch = ''
    substituteInPlace cmd/cmd_test.go --replace "development" "${version}"
  '';
```